04.ls/ls.rb

```diff
@@ -1,30 +1,53 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-NUMBER_OF_COL = 3
+require 'optparse'
+
+NUMBER_OF_COL_MAX = 3
 SPACE_WIDTH = 2
 
-def get_filenames(target_path)
-  Dir.glob('*', base: target_path)
+def search_option(argv)
+  opt = OptionParser.new
+  option_params = {}
+  opt.on('-a') { |v| option_params[:a] = v }
+  arguments_without_options = opt.parse(argv)
+  return option_params, arguments_without_options
```

Comment:
```suggestion
  [option_params, arguments_without_options]
```

rubocopの指摘はこれで直るはずですー。

Reply:
ありがとうございます。とりあえず配列で値を返すように変更しました。
https://github.com/cellotak/ruby-practices/pull/7#discussion_r1286663733
の後半で相談させて頂いた内容についてはいかがでしょうか。(メソッドを分けるべきかどうか)